src/Build/Evaluation/Evaluator.cs

```diff
@@ -626,7 +626,7 @@ private void Evaluate()
                     }
                 }
 
-                _evaluationLoggingContext.LogProjectEvaluationStarted();
+                _evaluationLoggingContext.LogProjectEvaluationStarted(_data.GlobalPropertiesDictionary[MSBuildConstants.MSBuildIsRestoring] is not null); ;
```

Comment:
We currently set the property only to `true` but it's not unconceivable that at some point someone would want to set it to `false` to mean "we're not restoring". For future proofing, would you consider parsing the value instead of just checking that the prop is defined?